Buildings/Experimental/RadiantControl/Validation/ConPluLoc.mo

```diff
@@ -0,0 +1,68 @@
+within Buildings.Experimental.RadiantControl.Validation;
+model ConPluLoc "Validation model for radiant control"

```

Comment:
Again, please use full words for model names, each word with and uppercase first letter. This applies to model and block names, whereas their instances have each a name composed of the first each word 3-letter fragments in camel case.